scripts/assistant/add_model.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+set -e
+
+function add_model() {
+  [ -z "$SESSION_TOKEN" ] && SESSION_TOKEN_VALUE="" || SESSION_TOKEN_VALUE='"session_token": "'$SESSION_TOKEN'"'
+  [ -z "$SESSION_TOKEN_VALUE" ] && echo "[ User session disabled ]"
+
+  ENDPOINT=https://${USERNAME}:${PASSWORD}@${BIND_ADDRESS}:${BIND_PORT}
+
+  curl -s -k "${ENDPOINT}/_cluster/settings" -XPUT -H 'Content-Type: application/json' -d '{
+    "persistent" : {
+      "plugins.ml_commons.trusted_connector_endpoints_regex":
+      [ "^https://runtime\\.sagemaker\\..*[a-z0-9-]\\.amazonaws\\.com/.*$",
+        "^https://api\\.openai\\.com/.*$",
+        "^https://api\\.cohere\\.ai/.*$",
+        "^https://bedrock-runtime\\.[a-z0-9-]+\\.amazonaws\\.com/.*$"
+      ]
+    }
+  }' | jq
+
+  # shellcheck disable=2016
+  CONNECTOR=$(curl -s -k "${ENDPOINT}/_plugins/_ml/connectors/_create" -XPOST -H 'Content-Type: application/json' -d '{
+    "name": "BedRock test claude Connector",
+    "description": "The connector to BedRock service for claude model",
+    "version": 1,
+    "protocol": "aws_sigv4",
+    "parameters": {
+        "region": "'$REGION'",
+        "service_name": "bedrock",
+        "anthropic_version": "bedrock-2023-05-31",
+        "endpoint": "bedrock.'$REGION'.amazonaws.com",
+        "auth": "Sig_V4",
+        "content_type": "application/json"
+    },
+    "credential": {
+      "access_key": "'$ACCESS_KEY_ID'",
+      "secret_key": "'$SECRET_ACCESS_KEY'",
+      '"$SESSION_TOKEN_VALUE"'
```

Comment:
i think this will fail, if `$SESSION_TOKEN_VALUE` is empty, then there's an extra comma after `secret_key`. `jq` or include the comma in the variable might help, or add a TODO and i'll fix this later

Reply:
yeah, i was thinking about that and verified that that it doesn't cause any issue would occur with sending a request with an unneeded  `,` i didn't run into any issue but it could be OS specific. do you think we should add something there or prefer the prepend to the variable ?

Reply:
hm standard json parser shouldn't allow this, but if it's working fine in opensearch then it's ok